common/cmake/devices.cmake

```diff
@@ -586,6 +599,15 @@ function(DEFINE_DEVICE)
     # Generate a rr_graph for a device.
     #
 
+    # Use the device specific channel with for the virtual graph if provided.
+    # If not use a dummy value (assuming that the graph will get patched
+    # anyways).
+    if("${DEFINE_DEVICE_ROUTE_CHAN_WIDTH}" STREQUAL "")
+      set(RRXML_VIRT_ROUTE_CHAN_WIDTH 6) # FIXME: Where did the number come from?
```

Comment:
This value was the smallest value that could consistently route a wire.  Largest numbers result in large virtual graphs, which take more time to generate and write/read from disk.  An even number was chosen because it is required for BIDIR channel definitions.

Reply:
Thanks for the explanation. I'll include it in a comment

Reply:
I don't see the comment.  Did you forget to push?